src/cli/install.sh

```diff
@@ -224,6 +224,50 @@ zsh)
         done
     fi
     ;;
+bash)
+    commands=(
+        "export $install_env=$quoted_install_dir"
+        "export PATH=$bin_env:\$PATH"
+    )
+
+    bash_configs=(
+        "$HOME/.bashrc"
+        "$HOME/.bash_profile"
+        "$XDG_CONFIG_HOME/.bash_profile"
+        "$XDG_CONFIG_HOME/.bashrc"
+        "$XDG_CONFIG_HOME/bash_profile"
+        "$XDG_CONFIG_HOME/bashrc"
+    )
```

Comment:
`$XDG_CONFIG_HOME` can be undefined
```suggestion
    )

    if [[ ${XDG_CONFIG_HOME:-} ]]; then
        bash_configs+=(
            "$XDG_CONFIG_HOME/.bash_profile"
            "$XDG_CONFIG_HOME/.bashrc"
            "$XDG_CONFIG_HOME/bash_profile"
            "$XDG_CONFIG_HOME/bashrc"
        )
    fi
```